tests/docker/instance_activate_ipsec_lb_agent_resp

```diff
@@ -28,7 +28,12 @@
                                 "Type": "tcp",
                                 "PrivatePort": 8080
                             }
-                        ]
+                        ],
+                        "Labels": {
```

Comment:
Minor: is there a reason why we distinguish between system container types? If all system containers require the same processing based on the label, would putting true/false suffice?